controllers/usercontroller.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable */
```

Comment:
please don't disable eslint try to fix the errors for our code should be uniform